pyseir/ensembles/ensemble_runner.py

```diff
@@ -102,16 +102,16 @@ def __init__(
 
         county_fips = None if self.agg_level is AggregationLevel.STATE else self.fips
 
-        if not covid_timeseries:
-            # TODO(tom): deprecate all code paths that get here. I'm trying to move towards loading all data
-            # once and passing it around in parameters.
-            covid_timeseries = JHUDataset.local().timeseries()
-        else:
-            covid_timeseries = covid_timeseries.timeseries()
-
-        self.covid_data = covid_timeseries.get_data(
-            aggregation_level=self.agg_level, country="USA", state=self.state_abbr, fips=county_fips
-        ).sort_values("date")
+#         if not covid_timeseries:
```

Comment:
This was the patch I put in so that the build didn't run out of memory. I've fixed it in master. There will be a merge conflict and we can just delete this whole commented section.